Gems/ScriptCanvas/Code/Include/ScriptCanvas/AutoGen/ScriptCanvas_Macros.jinja

```diff
@@ -173,11 +173,10 @@ AZStd::tuple<{{returnTypes|join(", ")}}>
 
 {# ------- #}
 
-
 {# ------------------------------------------------------------------------------------- #}
 {# NODEABLES #}
 
-{# TODO-LS: This macro lacks parsing the parameters provided to a Contract tag #}
+{# TODO: This macro lacks parsing the parameters provided to a Contract tag #}
```

Comment:
Nitpick: TODO comments should be removed before submitting as these comments don't age well. It's better to add a ticket to the backlog to have the remaining work tracked properly.